envoy/http/alternate_protocols_cache.h

```diff
@@ -95,10 +95,14 @@ class AlternateProtocolsCache {
     virtual bool isHttp3Broken() const PURE;
     // Returns true if HTTP/3 is confirmed to be working.
     virtual bool isHttp3Confirmed() const PURE;
+    // Returns true if HTTP/3 has failed recently.
+    virtual bool hasHttp3FailedRecently() const PURE;
```

Comment:
You might consider making the alternate protocols cache changes in a stand-alone PR since they'll likely be simple.

Reply:
Will work on it.